lnwallet/commitment.go

```diff
@@ -36,11 +39,14 @@ type CommitmentKeyRing struct {
 	// the local HTLC base point. This value is needed in order to
 	// derive the final key used within the HTLC scripts in the commitment
 	// transaction.
+	//
+	// NOTE: This will always refer to "our" local HTLC key, regardless of
```

Comment:
Yeah, this led to some confusion during review. 

I also tried switching the perspective always, but that turned out to be a large change since it will ripple through many parts of the codebase, so felt it was better to leave it as-is for now, with some clarifying commentary.